recipes/python-asana/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-asana" %}
+{% set version = "0.10.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Asana/python-asana/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
I am getting the source from GitHub because the PyPI package is 8 versions out of date (see https://github.com/Asana/python-asana/issues/129).